src/qt/pivx/pivxgui.h

```diff
@@ -59,7 +59,7 @@ class PIVXGUI : public QMainWindow
     int getNavWidth();
 Q_SIGNALS:
     void themeChanged(bool isLightTheme, QString& theme);
-    void windowResizeEvent(QResizeEvent* event);
+    void windowResizeEvent();
```

Comment:
I wrote it in other place, writing it here too because it's the main signal.

We shouldn't remove the `QResizeEvent` argument and use it instead. `QResizeEvent`object contain the new and old sizes of the window. It's an useful object that will prevent duplicated calls to `window->width()` and `window->height()` inside the slots that are connected to the signal.

Reply:
I've reverted the change here, was originally done when a prior version of Qt that is now unsupported had an issue with the new connect syntax and `QResizeEvent`.

I don't intend to address the duplicated calls within the function in this PR though.